src/main/java/io/datareplication/model/snapshot/SnapshotIndex.java

```diff
@@ -34,11 +37,18 @@ public SnapshotIndex(@NonNull SnapshotId id, @NonNull Timestamp createdAt, @NonN
     }
 
     public @NonNull Body toJson() {
-        throw new RuntimeException("not implemented");
+        Gson gson = new GsonBuilder()
```

Comment:
I would like to implement only one GsonBuilder as field, including the complete codec (or as a singleton instance over the project)